_scripts/instruction-widget/install.js

```diff
@@ -251,6 +254,14 @@ module.exports = function(context) {
     context.dns_package_prefix = "python-certbot-dns";
   }
 
+  snap_install = function() {
+    template = "snap";
+    context.base_command = "certbot";
+    context.cron_included = true;
+    context.install_command = "sudo snap install --beta --classic";
+    context.package = "certbot";
+  }
```

Comment:
and I assume we don't have dns plugins ready?

Reply:
follow-up, what does it look like on the wildcard tab? is there a note about how snaps won't let you get a wildcard cert?

Reply:
found the answer to the former in the release announcement draft.

Reply:
The wildcard tab looks the same as other systems that don't have DNS plugins available. Here's a screenshot:
![Screen Shot 2020-04-21 at 3 38 45 PM](https://user-images.githubusercontent.com/6504915/79921061-516c7300-83e6-11ea-9de6-8ba31570fdf4.png)